odk1-src/central-install-digital-ocean.rst

```diff
@@ -0,0 +1,235 @@
+ - If your domain name has started working, you can visit it in a web browser to check that you get the ODK Central management website.
+
+Once we're finally sure that everything is working, run ``systemctl enable docker-compose@central``. This will make sure the ODK server is always running, even if something goes wrong or the machine reboots.
+
+You're almost done! All you have to do is create an Administrator account so that you can log into Central.
+
+.. _central-install-digital-ocean-account:
+
+Logging into ODK Central
+------------------------
+
+If visiting your server domain name address in your browser does not load the ODK Central management website, you may have to wait a few minutes or hours (possibly even a day) for the domain name itself to get working. These instructions are explained in further depth on the page detailing the :doc:`central-command-line`.
+
+Once you do see it working, you'll want to set up your first administrator account. To do this:
+
+ - Ensure that you are in the ``central`` folder on your server. If you have not closed your console session from earlier, you should be fine. If you have just logged back into it, you'll want to run ``cd central`` to navigate to that folder.
+ - Then, type ``docker-compose exec service odk-cmd --email YOUREMAIL@ADDRESSHERE.com user-create``, substituting your email address as appropriate. Press **Enter**, and you will be asked for a password for this new account.
+ - The previous step created an account but did not make it an administrator. To do this, type ``docker-compose exec service odk-cmd --email YOUREMAIL@ADDRESSHERE.com user-promote`` **Enter**.
+ - You are done for now, but if you ever lose track of your password, you can always reset it by typing ``docker-compose exec service odk-cmd --email YOUREMAIL@ADDRESSHERE.com user-set-password``. As with account creation, you will be prompted for a new password after you press **Enter**.
+
+Once you have one account, you do not have to go through this process again for future accounts: you can log into the website with your new account, and directly create new users that way.
+
+You're done! Congratulations. In the future, you may wish to consult the :doc:`central-upgrade` guide, but for now you may begin using ODK Central. The :doc:`central-using` sections can help you with your next steps if you aren't sure how to proceed.
+
+.. _central-install-digital-ocean-advanced:
+
+Advanced Configuration Options
+==============================
+
+The following sections each detail a particular customization you can make to your server setup. Most installations should not need to perform these tasks, and some of them assume some advanced working knowledge on administering Linux web servers. If you aren't sure what something means, the best option is probably to skip the section completely.
+
+.. _central-install-digital-ocean-custom-dkim:
```

Comment:
I'd say drop the custom. Since it's not custom DKIM, just regular old DKIM. I don't feel strongly about this.

Reply:
copy pasta